packages/loot-core/src/server/rules/condition.ts

```
@@ -348,6 +351,13 @@ export class Condition {
         }
         return String(fieldValue).indexOf(this.value) !== -1;
 
+      case 'hasAnyTag':
+        if (fieldValue === null) {
+          return false;
+        }
+        const tags = extractTagsForFilter(this.value);
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)